README.md

```diff
@@ -194,8 +197,10 @@ The format of this glossary is modeled after [HugoGiraudel/SJSJ](https://github.
 ### L
 
 * LCD:
-* LED:
-* Light-emitting diode:
+* **LED**:
+    See [Light-emitting diode](#light-emitting-diode)
+* **Light-emitting diode**<a name="light-emitting-diode"></a>:
+   a two-lead semiconductor which emits light when activated
```

Comment:
Can you indent this line by one more space to align with the previous definition?